### DISCUSSION TITLE
Magick++ Low level pixel access documentation is out of date

### DISCUSSION THREAD

**Discussion by planetmarshall** [ImageMagick/ImageMagick#6704] [Contribute]
http://www.imagemagick.org/Magick++/Image.html#Raw%20Image%20Pixel%20Access states that `Image::getPixels` returns a `PixelPacket*`, however the signature as of 7.05.3 is

```cpp
Magick::Quantum *Magick::Image::getPixels(const ssize_t x_,const ssize_t y_, const size_t columns_,const size_t rows_);
```

There don't appear to be any tests or examples using these methods.

**Comment by mikayla-grace** [comment]
We updated the Magick++ documentation.   It needs further edits and we will get to it as time permits.

**Comment by dlemstra** [comment]
We could use some help here.
